src/sentry/release_health/metrics_sessions_v2.py

```diff
@@ -44,19 +45,19 @@
 #: Referrers for snuba queries
 #: Referrers must be searchable, so no string interpolation here
 REFERRERS = {
-    "user": {
+    MetricKey.USER: {
```

Comment:
These two are not equivalent because this is an Enum, to achieve the same see comment above

Reply:
Should be fine because the only usage now also uses `MetricKey` as key: https://github.com/getsentry/sentry/blob/e350e6b614faf02a2ca1b42cb012c5f30e286da8/src/sentry/release_health/metrics_sessions_v2.py#L388